lib/systems/inspect.nix

```diff
@@ -91,7 +91,7 @@ rec {
     isMusl         = with abis; map (a: { abi = a; }) [ musl musleabi musleabihf muslabin32 muslabi64 ];
     isUClibc       = with abis; map (a: { abi = a; }) [ uclibc uclibceabi uclibceabihf ];
 
-    isEfi = [
+    hasEfi = [
```

Comment:
Can you add a comment here that explains the distinction?

Reply:
Done.